server/src/packages/pas-form/applicants/applicants.controller.ts

```diff
@@ -39,24 +61,37 @@ export class ApplicantsController {
     return {
       dcp_applicantinformationid: id,
       ...body,
-    }
+    };
   }
 
   @Post('/')
   create(@Body() body) {
-    const allowedAttrs = pick(body, APPLICANT_ATTRIBUTES);
+    const { target_entity } = body;
+  
+    let allowedAttrs;
+
+    // determine the appropraite attributes based on the entity type
+    if (target_entity === 'dcp_applicantinformation') {
+      allowedAttrs = pick(body, APPLICANT_ATTRIBUTES);
+    } else if (target_entity === 'dcp_applicantrepinformation') {
+      allowedAttrs = pick(body, APPLICANT_REPRESENTATIVE_ATTRIBUTES);
+    }
 
     if (body.pas_form) {
-      return this.crmService.create('dcp_applicantinformations', {
+      let applicantBody = {
         ...allowedAttrs,
+      }
+      const objKey = `dcp_${target_entity}_dcp_pasform@odata.bind` 
+      // Dy365 syntax for associating a newly-created record
+      // with an existing record.
+      // see: https://docs.microsoft.com/en-us/powerapps/developer/common-data-service/webapi/create-entity-web-api#associate-entity-records-on-create
+      applicantBody[objKey] = [`/dcp_pasforms(${body.pas_form})`]
 
-        // Dy365 syntax for associating a newly-created record
-        // with an existing record.
-        // see: https://docs.microsoft.com/en-us/powerapps/developer/common-data-service/webapi/create-entity-web-api#associate-entity-records-on-create
-        'dcp_dcp_applicantinformation_dcp_pasform@odata.bind': [`/dcp_pasforms(${body.pas_form})`],
-      });
+      console.log(applicantBody);
+      
+      return this.crmService.create(`${target_entity}s`, applicantBody);
```

Comment:
Currently getting this error when trying to `Post`:
![image](https://user-images.githubusercontent.com/5316367/82236759-9363d880-9902-11ea-910f-eaabfb4f5888.png)

Reply:
Tried to `post` without the pluralization, received same error
![image](https://user-images.githubusercontent.com/5316367/82238094-b7c0b480-9904-11ea-905b-ff70d0364419.png)

Reply:
This is the error when using `dcp_applicantrepresentativeinformation` (passed in through frontend action), which is what shows in the CRM power pane, instead of the shortened `dcp_applicantrepinformaiton` found in the crm metadata.
![image](https://user-images.githubusercontent.com/5316367/82238293-0e2df300-9905-11ea-991c-aba411290750.png)